src/pybpmn/parser.py

```diff
@@ -220,6 +218,18 @@ def scale_anns_to_img_width_(self, anns: List[Annotation], bpmn_path: Path, img:
                 a.tail = a.waypoints[0]
                 a.head = a.waypoints[-1]
 
+        arrow_min_wh_scaled = self.arrow_min_wh * max(img.size) / self.img_max_size_ref
+        edge_anns = filter(lambda a: a.category in syntax.BPMNDI_EDGE_CATEGORIES, anns)
```

Comment:
Nit: In Python werden list comprehensions ggü funktionaler Programmierung bevorzugt, also in diesem Beispiel `edge_anns = [a for a in anns if a.category in syntax.BPMNDI_EDGE_CATEGORIES]`